docs/Introduction/tutorials/intermediates-tutorial.md

```diff
@@ -218,63 +294,64 @@ function getHouseName(uint256 id) private pure returns (string memory) {
     <a href="../deploy-your-first-contract/" title="">What is Remix?</a>
 </div>
 
-# 5. Deployment
+# 5. How do I deploy to testnet?s
 
-Time to compile and deploy the contract! If you don't know how to deploy a contract to the Kovan testnet from Remix, follow **[The Basics tutorial](/docs/beginners-tutorial)**.
-
-This deployment is slightly different than the example from the beginners tutorial. In this tutorial, we have to pass in parameters to the constructor upon deployment.
+You will now deploy your completed contract. This deployment is slightly different than the example in the [The Basics: Using Data Feeds](/docs/beginners-tutorial/#7-how-do-i-deploy-to-testnet) tutorial. In our case, you will have to pass in parameters to the constructor upon deployment.
 
 Once compiled, you'll see a menu that looks like this in the deploy pane:
 
 ![Remix Deployed Contract](/files/f6c0c2b-Screenshot_2020-12-18_at_16.23.19.png)
 
-Click the caret arrow on the right hand side of "Deploy" to expand the parameter fields, and paste the following values in:
+Click the caret arrow on the right hand side of **Deploy** to expand the parameter fields, and paste the following values in:
 
 - `0xdD3782915140c8f3b190B5D67eAc6dc5760C46E9`
 - `0xa36085f69e2889c224210f603d836748e7dc0088 `
 - `0x6c3699283bda56ad74f6b855546325b68d482e983852a7a82979cc4807b641f4`
 - `100000000000000000`
 
-These are the coordinator address, LINK address, key hash, and fee. Click deploy and use your Metamask account to confirm the transaction.
+These are the coordinator address, LINK address, key hash, and fee. For a full reference of the addresses, key hashes, and fees for each network, see [VRF Contracts](../vrf-contracts/). Click deploy and use your Metamask account to confirm the transaction.
 
-> 📘 Address, Key Hashes and more
->
-> For a full reference of the addresses, key hashes and fees for each network, see [VRF Contracts](../vrf-contracts/).
+**Note**: You should [have some Kovan ETH](/docs/beginners-tutorial/#obtaining-testnet-eth) in your Metamask account to pay for the GAS.
 
-(Note: you should <a href="/docs/beginners-tutorial/#obtaining-testnet-eth" target="_blank">have some Kovan ETH in your Metamask account</a> to pay for the GAS).
+At this point, your contract should be successfully deployed. However, it can't request anything yet since it doesn't own LINK. If you click `rollDice` with no LINK, the transaction will revert.
 
-Once deployed, the contract is almost ready to go! However, it can't request anything yet, since it doesn't own LINK. If we hit `rollDice` with no LINK, the transaction will revert.
+# 6. How do I obtain testnet LINK?
 
-# 6. Obtaining testnet LINK
+Since the contract is on testnet, as with Kovan ETH, you don't need to purchase *real* LINK. Testnet LINK can be requested and obtained from a [faucet](../link-token-contracts/).
```

Comment:
```suggestion
Because the contract is on testnet, as with Kovan ETH, you don't need to purchase *real* LINK. You can request and obtain Testnet LINK from a [faucet](../link-token-contracts/).
```